### PR TITLE
Print more verbose error messages for bosh.rb

### LINKED ISSUE #14: Output "raw_json" in a formatted way so the error message is visible
https://github.com/anynines/unit-tests-utils/blob/b64b73a490ff023d5a549f0b33fd075081286d98/lib/unit_tests_utils/bosh.rb#L58

### PR DESCRIPTION
**Pull request by kfkonrad**
fixes #14, but in a generic way for every call to the BOSH cli

### CODE REVIEW

File: lib/unit_tests_utils/bosh.rb [hunk 1]
```diff
@@ -158,5 +124,14 @@ def self.parse_json_from_create_release(raw_json)
     }
   end
 
+  def self.execute_or_raise_error(command, msg)
```
Comment (jebreuer) [inline lib/unit_tests_utils/bosh.rb]: Please implement tests for the function.
See: `spec/lib/unit_tests_utils/bosh_spec.rb`

File: spec/lib/unit_tests_utils/bosh_spec.rb [hunk 2]
```diff
@@ -272,4 +284,26 @@ def bosh_release_output(release_name, release_version)
       end
     end
   end
+
+  describe ".execute_or_raise_error" do
+    context "when the command returns a zero status" do
+      it "does not raise a BOSH exception" do
+        expect(UnitTestsUtils::Bosh).to receive(:execute_or_raise_error).once.
```
Comment (gogolok) [inline spec/lib/unit_tests_utils/bosh_spec.rb]: Hmm, am I wrong or are you just testing nothing here?

The class function `UnitTestsUtils::Bosh.execute_or_raise_error` should be tested and not be replaced by rspec magic.

Just provoke an error with a simple shell command:
```ruby
UnitTestsUtils::Bosh.execute_or_raise_error("ls /none-existing-dir", "the-msg")
# stdout, stderr, exit_status = Open3.capture3("ls /none-existing-dir")
# => ["", "ls: /none-existing-dir: No such file or directory\n", #<Process::Status: pid 78821 exit 1>]
```

And then have a successful run:
```ruby
UnitTestsUtils::Bosh.execute_or_raise_error("echo works", "works")
# irb(main):002:0>  stdout, stderr, exit_status = Open3.capture3("echo works")
# => ["works\n", "", #<Process::Status: pid 78812 exit 0>]
```

Comment (jebreuer) [reply spec/lib/unit_tests_utils/bosh_spec.rb]: You can also use `/usr/bin/env true` for a successful command and `/usr/bin/env false` for a failing one.

File: lib/unit_tests_utils/bosh.rb (hunk 1 above)
Comment (gogolok) [reply lib/unit_tests_utils/bosh.rb]: Done

**Pull request merged**